src/components/LoginButton.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import firebase from 'firebase';
+import Button from '@material-ui/core/Button';
+import LoginDialog from './LoginDialog';
+
+export default function LoginButton() {
+  const { currentUser } = firebase.auth(); // determines how to initialize isSignedIn
+  const [isSignedIn, setLoggedIn] = useState(!!currentUser);
+  const [loginDialogOpen, setDialogOpen] = useState(false); // initialized to false
```

Comment:
Please fix these to use the `const [X, setX]` naming pattern seen throughout the [React docs](https://reactjs.org/docs/hooks-state.html).

Reply:
Fixed

Reply:
Generally speaking, it's also not good practice to mark a PR comment as fixed unless the fix can actually be seen in the code on here

Reply:
*changed